lib/wash_out/version.rb

```diff
@@ -1,3 +1,3 @@
 module WashOut
-  VERSION = "0.9.2"
+  VERSION = "1.1.1"
```

Comment:
I moved to 1.0.0 first but then made a few more changes hence internally changed that version to 1.1.1. But can pick a different version based your convention for major minor.